include/controller.h

```diff
@@ -53,6 +53,8 @@ class controller {
 		return refresh_on_start;
 	}
 	void enqueue_url(const std::string& url,
+        const std::string& title,
+        const time_t& pubDate,
```

Comment:
`time_t` is [a simple type](https://en.cppreference.com/w/c/chrono/time_t)—probably a `long` or a `double`—so it can be passed by value without any performance penalty. So I'd remove that reference.